tools/distpartitioning/convert_partition.py

```diff
@@ -383,68 +426,41 @@ def create_dgl_object(
             [edge_id_start, edge_id_start + np.sum(etype_ids == etype_id)]
         )
         edge_id_start += np.sum(etype_ids == etype_id)
-    memory_snapshot("CreateDGLObj_UniqueNodeIds: ", part_id)
 
-    # get the edge list in some order and then reshuffle.
-    # Here the order of nodes is defined by the sorted order.
     uniq_ids, idx, part_local_src_id, part_local_dst_id = _get_unique_invidx(
         shuffle_global_src_id,
         shuffle_global_dst_id,
         np.arange(shuffle_global_nid_range[0], shuffle_global_nid_range[1] + 1),
     )
+    shuffle_global_src_id = None
+    shuffle_global_dst_id = None
 
+    # inner nodes
```

Comment:
This comment provides zero additional information than the code itself.
Please consider removing it or elaborate more.